units.libs.lua

```diff
@@ -183,7 +183,7 @@ StaticLibrary {
 		Filters = {
 			{ Pattern = "[/\\]windows[/\\]"; Config = { "win32-*", "win64-*" } },
 			{ Pattern = "[/\\]macosx[/\\]"; Config = "mac*-*" },
-			{ Pattern = "[/\\]x11[/\\]"; Config = { "x11-*" } },
+			{ Pattern = "[/\\]X11[/\\]"; Config = { "x11-*" } },
```

Comment:
No idea really. This is a clean debian install and I just installed git and made no changes at all to it so I'm not sure what the cause of it would be.